e2e/cypress/integration/enterprise/system_console/cluster_spec.js

```diff
@@ -50,4 +50,31 @@ describe('Cluster', () => {
             cy.get('#EnableExperimentalGossipEncryptiontrue').should('have.attr', 'checked');
         });
     });
+
+    it('Can change Gossip Compression', () => {
+        cy.findByTestId('EnableGossipCompression').scrollIntoView().should('be.visible').within(() => {
+            // * Verify that setting is visible and matches text content
+            cy.get('.control-label').should('be.visible').and('have.text', 'Enable Gossip compression:');
+
+            // * Verify that the help setting is visible and matches text content
+            const contents = 'When true, all communication through the gossip protocol will be compresssed. It is recommended to keep this flag disabled.';
+            cy.get('.help-text').should('be.visible').and('have.text', contents);
+
+            // * Verify that Gossip Compression is set to true by default
+            cy.get('#EnableGossipCompressiontrue').should('have.attr', 'checked');
+        });
+
+        // # Disable Gossip Compression
+        cy.apiUpdateConfig({ClusterSettings: {
+            Enable: true,
+            UseExperimentalGossip: true,
+            EnableGossipCompression: false,
+        }});
```

Comment:
Minor/stylistic, but I think it'd be clearer unwrapped 0.5/5
```suggestion
        cy.apiUpdateConfig({
            ClusterSettings: {
                Enable: true,
                UseExperimentalGossip: true,
                EnableGossipCompression: false,
            }
        });
```

Reply:
We should really be either auto-formatting these things, or flagging them via a linter.